rtgui/locallabtools.h

```diff
@@ -657,13 +702,18 @@ class LocallabShadow:
     void updateMaskBackground(const double normChromar, const double normLumar, const double normHuer, const double normHuerjz) override;
 
     void shMethodChanged();
+    void ghsMethodChanged();
     void inversshChanged();
+    void ghs_smoothChanged();
+    void ghs_invChanged();
     void showmaskSHMethodChanged();
     void showmaskSHMethodChangedinv();
     void enaSHMaskChanged();
 
     void updateShadowGUI1();
     void updateShadowGUI2();
+    void updateShadowGUI3();
```

Comment:
Give these functions more descriptive names. It's not clear what the differences between them are.

Reply:
I change for 
updateShadowGUImask(); 
updateShadowGUIshmet();
updateShadowGUIsym();